{
  if (c != null) {
    try {
      c.close();
    }
 catch (    IOException ie) {
      LOG.info("Ignoring exception during close for " + c,ie);
    }
  }
}
